### PR TITLE
119 fix various issues

### LINKED ISSUE #119: Fix some issues
The following are all issues brought up by Richard:

- `The person [who is first and last], [who died], [and who became alive again] sends these words to you(followers) _implicit.`
Commas and coordinate relative clauses are blocking the agent from being recognized.

- `God kindly/graciously and powerfully caused [me(Paul) to serve God].`
The coordinate Adverbs are blocking the agent from being recognized

- `I(Jesus) know [some people say bad and false things about you(followers)].`
Coordinate attributive adjectives are not being recognized properly.

- `You(people) will become-J a house of the Spirit.`
The theta grid for become-J is empty. We should treat every argument as optional in that case, rather than treating every argument as extra, as it is currently doing

- `People did not learn about those things-D in earlier _adj times/generations.`
The usage of early-A is all underscores. We should treat every argument as optional in that case, rather than treating every argument as extra, as it is currently doing

- `(footnote) You(person) (imp) see Jeremiah 31:31-34.`
The hyphen in the reference is causing errors

- We should allow an underscore comment after the last period.

### PR DESCRIPTION
**Pull request by craigp-atw**
Resolves #119 . Refer to that issue for details of what the issues were.

- `The person [who is first and last], [who died], [and who became alive again] sends these words to you(followers) _implicit.`
image

- `God kindly/graciously and powerfully caused [me(Paul) to serve God].`
image

- `I(Jesus) know [some people say bad and false things about you(followers)].`
image

- `You(people) will become-J a house of the Spirit.`
image

- `People did not learn about those things-D in earlier _adj times/generations.`
image

- `(footnote) You(person) (imp) see Jeremiah 31:31-34.`
image

- Allow underscore comments after the last period.
image
image

### CODE REVIEW

File: app/src/lib/rules/part_of_speech_rules.js [hunk 1]
```diff
@@ -275,6 +275,36 @@ const part_of_speech_rules_json = [
...
+	{
+		'name': 'If Adjective-Adposition "about" followed by an Adjective, delete the Adposition',
+		'category': 'Adjective|Adposition',
+		'trigger': { 'stem': 'about' },
+		'context': {
+			'followedby': { 'category': 'Adjective' },
+			'notfollowedby': { 'stem': 'all|many|each' },
+		},
+		'remove': 'Adposition',
+		'comment': "Daniel 2:1  \"for about(Adj/Adp) two years\"  This rule applies specifically to the word \"about\" because in general, both Adjectives and Adpositions can precede Adjectives.  Exclude the word 'all' as in Ruth 3:16 \"Ruth told Naomi about(Adj/Adp) all(Adj) the things that Boaz did.\" Exclude 'many' as in Kande's Story 5:9 \"They talked about many things.\"  This rule really should specify that when 'about' is followed by a numeric Adjective, delete the Adposition.",
```
Comment (longrunningprocess) [inline app/src/lib/rules/part_of_speech_rules.js:296]: swap those quotes to get rid of the backslashes

File: app/src/lib/rules/part_of_speech_rules.js [hunk 2]
```diff
@@ -275,6 +275,36 @@ const part_of_speech_rules_json = [
...
+	},
+	{
+		'name': 'If Adjective-Adposition followed by a Noun, delete the Adjective',
+		'category': 'Adjective|Adposition',
+		'trigger': { 'stem': 'about' },
+		'context': {
+			'followedby': { 'category': 'Noun' },
+		},
+		'remove': 'Adjective',
+		'comment': "Daniel 2:1 \"Nebuchadnezzar worried about(Adj/Adp) this dream.\"  The Adjective 'about' means 'approximately' which can't be followed by a Noun.",
```
Comment (longrunningprocess) [inline app/src/lib/rules/part_of_speech_rules.js:306]: swap those quotes to get rid of the backslashes

**Review by longrunningprocess — CHANGES_REQUESTED** [review]
excellent!  I noticed a couple of minor things, you can merge after you fix them.

File: app/src/lib/rules/part_of_speech_rules.js (hunk 1 above)
Comment (craigp-atw) [reply app/src/lib/rules/part_of_speech_rules.js:296]: I copied that comment from TBTA, and it contains both single and double quotes. So either I backslash the double quotes, or I backslash the single quotes. Although I could remove some of them altogether

Comment (craigp-atw) [reply app/src/lib/rules/part_of_speech_rules.js]: I changed all the string to only contain single quotes, and left the outer string quotes as double.

**Pull request merged**